lte/gateway/c/core/oai/tasks/amf/amf_security_mode_control.cpp

```diff
@@ -122,11 +122,11 @@ nas_amf_smc_proc_t* nas5g_new_smc_procedure(amf_context_t* const amf_context) {
  ***************************************************************************/
 static int amf_security_request(nas_amf_smc_proc_t* const smc_proc) {
   OAILOG_FUNC_IN(LOG_NAS_AMF);
-  ue_m5gmm_context_s* ue_mm_context = NULL;
-  amf_context_t* amf_ctx            = NULL;
-  amf_sap_t amf_sap                 = {};
-  int rc                            = RETURNerror;
-  smc_proc->T3560.id                = NAS5G_TIMER_INACTIVE_ID;
+  std::shared_ptr<ue_m5gmm_context_t> ue_mm_context;
+  amf_context_t* amf_ctx = NULL;
```

Comment:
```suggestion
  amf_context_t* amf_ctx = nullptr;
```

Reply:
addressed review comment, resolved all occurrences in same fils